index.js

```diff
@@ -22,7 +27,16 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
```

Comment:
Is `this.options` intentionally assigned to `this.options || {}` or is this a "typo"?